### DISCUSSION TITLE
When I switch desktops, the transparency setting will become invalid

### DISCUSSION THREAD

**Discussion by summerxiaohao** [WindowTop/WindowTop-App#396] [General]
![Snipaste_2024-11-22_10-09-59](https://github.com/user-attachments/assets/e97c42dc-6084-4a4a-a744-a38b9b8c2a44)
When I set up and use two desktops in the Windows system, I set the transparency and top setting for a program on Desktop 1. If I switch to Desktop 2 and then back to Desktop 1, after repeating this several times, the transparency I set for the program on Desktop 1 will become invalid and return to its default value.
My version: v5.25.1
-----------------------------------------
![Snipaste_2024-11-22_10-19-39](https://github.com/user-attachments/assets/ba0e8469-15a9-4b0f-9bb7-ec703e138f54)
-----------------------------------------
![Snipaste_2024-11-22_10-19-59](https://github.com/user-attachments/assets/3c6b6edd-8801-4f07-b0b5-4afb4f526ba7)

**Comment by gileli121** [comment]
@summerxiaohao  

Thank you for the report.  
This is an interesting and unexpected bug.  

I was able to reproduce it very easily and will work on a fix, likely today or the day after.  

I’ve also opened a bug issue based on this discussion:  
[https://github.com/WindowTop/WindowTop-App/issues/397](https://github.com/WindowTop/WindowTop-App/issues/397)

**Comment by gileli121** [comment]
@summerxiaohao  
I’ve fixed the issue. Can you please test it and let me know?

## Download  
- **Installed version**: [Download setup](https://github.com/WindowTop/WindowTop-App/releases/download/5.25/WindowTop.5.25.2.-.setup.exe)
- **Portable version**: [Download portable](https://github.com/WindowTop/WindowTop-App/releases/download/5.25/WindowTop.v5.25.2.Portable.zip)

Thank you
